cmake/Modules/OpmSatellites.cmake

```diff
@@ -1,4 +1,8 @@
 # - Build satellites that are dependent of main library
+
+option(ADD_DISABLED_CTESTS "Add the tests which are disabled due to failed preconditions to the ctest output (this makes ctest return an error if such a test is present)" ON)
+mark_as_advanced(ADD_DISABLED_CTESTS)
```

Comment:
same feeling here: I've no really strong feelings about this, I just think that in this regard "less is more", i.e. only the options which most users are likely to care about should be displayed by default.